ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/tasks/pre_workload.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: virtualenv path to use for pip
+  set_fact:
+    pip_path: "{{ ansible_python_interpreter | dirname }}"
+
+
+- name: Install lxml into virtualenv
+  command:  "{{ pip_path }}/pip install lxml"
+  become: True
+
+- name: Run shell 'oc' command to retrieve the cluster subdomain
+  shell: "oc get routes -n openshift-console --field-selector=spec.to.name==console -o jsonpath='{.items[].spec.host}' | cut -d'.' -f 2- | cut -f 1"
```

Comment:
Use `kubernetes.core.k8s_info`. See Let's Encrypt or Gitea workloads for an example.